android/src/com/mapswithme/maps/search/SearchAdapter.java

```diff
@@ -176,6 +176,11 @@ else if (!TextUtils.isEmpty(result.description.roadShields))
         }
       }
 
+      if (result.isHotel && result.stars != 0)
+      {
+        tail.append(" • ").append("★★★★★★★".substring(0, Math.min(7, result.stars)));
```

Comment:
Let's link 7 to osm docs (maybe also create a constant).

Reply:
We already have a constant in cpp.
There is no OSM links about that, just knowledge.

Reply:
It makes sense to also link it with the generator that should strip the wrong amount of stars.